pkg/engine/exceptions.go

```diff
@@ -53,6 +55,20 @@ func matchesException(
 		resource = policyContext.OldResource()
 	}
 	for _, candidate := range candidates {
+		if rule.HasVerifyImages() || rule.HasValidatePodSecurity() {
+			if candidate.Spec.HasImages() {
```

Comment:
I believe it is better to first check if policy exception has images or not, and if so then we check if rule has either verify images or validate.podSecurity.

Reply:
On it.